cleanlab/multilabel_classification/dataset.py

```diff
@@ -0,0 +1,93 @@
+import pandas as pd
+import numpy as np
+from cleanlab.filter import _find_multilabel_issues_per_class
+from cleanlab.internal.multilabel_utils import get_onehot_num_classes
+from collections import defaultdict
+
+
+def common_multilabel_issues(
+    labels=list,
+    pred_probs=None,
+    *,
+    class_names=None,
+    confident_joint=None,
+) -> pd.DataFrame:
+    """Returns the Tags that are often mis-labeled.
+
+
+    This method provides two scores in the Pandas DataFrame that is returned:
+
+    * **num_examples**: The number of examples where the given/predicted label differ.
+    * **Issue_probability**: `(num_examples / total number of examples in the dataset`).
+
+    This method works by providing any one (and only one) of the following inputs:
+
+    1. ``labels`` and ``pred_probs``, or
+    2. ``confident_joint``
+
+    Only provide **exactly one of the above input options**, do not provide a combination.
+
+    Parameters
+    ----------
+    labels : List[List[int]]
+        Refer to documentation for this argument in filter._find_multilabel_issues_per_class() for further details.
+
+    pred_probs : np.ndarray, optional
+      Refer to documentation for this argument in filter._find_multilabel_issues_per_class() for further details.
+
+
+    class_names : Iterable[str]
+        A list or other iterable of the string class names. The list should be in the order that
+        matches the label indices. So if class 0 is 'dog' and class 1 is 'cat', then
+        ``class_names = ['dog', 'cat']``.
+
+    Returns
+    -------
+    common_multilabel_issues : pd.DataFrame
+        Pandas DataFrame with indexed on Tag with columns "In_Given_Label", "In_Suggested_Label", "num_examples",
+        "Issue_probability"and a description of each below.
+        Each row corresponds to a pair of classes.
+
+        * *In_Given_Label*: Bool True/False
+        * *In_Suggested_Label*: Bool True/False
+        * *num_examples*: estimated number of issues for a different label other than the given label
+        * *Issue_probability*: the *num_examples* divided by the number of examples in the dataset.
+
+        By default, the DataFrame is ordered by "Issue_probability" descending.
```

Comment:
```suggestion
        DataFrame where each row corresponds to a Tag (specified as the row-index) with columns "In_Given_Label", "In_Suggested_Label", "Num_Examples", "Issue_Probability".

        * *In_Given_Label*: specifies whether the Tag is True/False in the given label  
        * *In_Suggested_Label*: specifies whether the Tag is  True/False in the suggested label (based on model prediction)
        * *Num_Examples*: Estimated number of examples with a label issue where this Tag is True/False as specified In_Given_Label but cleanlab suggests it should be as specified In_Suggested_Label. I.e. the number of examples in your dataset where the Tag was labeled as True but likely should have been False (or vice versa).
        * *Issue_Probability*: This is the  *Num_Examples* column divided by the total number of examples in the dataset. It corresponds to the relative overall frequency of each type of label issue in your dataset.

        By default, the rows in this DataFrame are ordered by "Issue_probability" descending.
```

Reply:
note my docstring has several changes. Make sure you update the code to reflect them and make sure the code actually does what the docstring says.

Reply:
"""make sure Num_Examples column only counts the number of examples flagged with a label issue where Tag = True/False in the given label, but takes opposite value in the predicted label from model."""

I'm letting Cleanlab determine this to be consistent, I could do this directly from Pred_probs if you'd like, 

true_but_false_count = sum(np.logical_and(label == 1, binary_label_issues))
false_but_true_count = sum(np.logical_and(label == 0, binary_label_issues))

I think this is correct